langroid/language_models/openai_gpt.py

```diff
@@ -288,6 +306,9 @@ class OpenAIGPTConfig(LLMConfig):
     chat_model: str = defaultOpenAIChatModel
     completion_model: str = defaultOpenAICompletionModel
     run_on_first_use: Callable[[], None] = noop
+    parallel_tool_calls: Optional[bool] = None
+    supports_json_schema: Optional[bool] = None
+    supports_strict_tools: Optional[bool] = None
```

Comment:
Please comment above this line on what `strict` does -- it is clear to me it is about strict decoding, but what happens if this is False? Does it mean:
- JSON will be produced but may not adhere exactly to the schema, OR
- JSON is not even guaranteed?

I'm thinking it's the former? Maybe make these config settings in the docs.

Reply:
Added comments on what this does; however, I'm not positive whether OpenAI uses basic JSON mode for tool calls without strict. Failures to produce JSON are at least rare